doc/dsdr/0010-placement-modelling.md

```diff
@@ -0,0 +1,105 @@
+
+**Age out model**
+
+To go some way towards mitigating this problem we make a simplifying assumption: all children who leave care after the age of 17 _actually_ leave care on their 18th birthday. This sounds like a gross oversimplification, but in fact most children who leave care aged 17 do leave on their 18th birthday (at least for the LAs we have studied so far). This is a useful assumption because it allows us to break the problem into two steps: what's the probability child x will stay in care until after the age of 17, and (if they don't) what will their duration in care be? If child x remains in care until after 17, then we know what their duration in care will be: it'll be whatever takes them up to their 18th birthday. If they don't, then we can use survival analysis to determine their duration in care for durations up to their 17th birthday. 
+
+Supporting this extra model requires an additional model input: the probability of exceeding 17 years of age in care by admission age. This is close to 0 for zero-age joiners, and exactly 1 for age 17 joiners. It's contained in the file `age-out-proportions.csv`.
+
+**Candidates files**
+
+Generate candidates process outputs 4 files: `simulated-candidates.csv`, `projected-candidates.csv`, `projected-age-out-candidates.csv`, `simulated-age-out-candidates.csv`.
+
+The `simulated-*` candidates files are organised by admission age and are used for generating new joiners, the `projected-*` files are indexed by period id and are used for closing open cases.
+The `*-age-out-candidates` files contain the results of attempting to generate closed cases which persist until 18, whereas the other files will generally not contain age out candidates. This is not _always_ the case: we will have been unable to generate projected-age-out candidates for some children (if there is nothing in the source data which allows us to chain a sequence which ends at 18 for that child). Likewise there will be age 17 joiners in the non-age-out files who age out by definition.
+
+**Projection**
```

Comment:
This section could use a couple of flow charts, but is still pretty understandable.